controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaDataTests.java

```diff
@@ -709,7 +708,7 @@ public void testGetScalingEvents() throws Exception {
         Response response = client.target(resourceURI).queryParam("from", fromDateTime).
                 queryParam("to", toDateTime).request().buildGet().invoke();
         assertEquals("Get Scaling Events response code", 200, response.getStatus());
-        final List<ScaleMetadata> scaleMetadataListResponse = response.readEntity(new GenericType<List<ScaleMetadata>>(){});
+        final List<ScaleMetadata> scaleMetadataListResponse = response.readEntity(new GenericType<List<ScaleMetadata>>() { });
```

Comment:
I had made these changes to fix checkstyle errors. I think, once I pull changes from master into my branch, these changes would go away, as these problems have been fixed in master now.